tasks-by-users/Rehan-Khan-Engineer/04-card-hover-effect/style.css

```diff
@@ -0,0 +1,111 @@
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  box-sizing: border-box;
+
+  font-family: "Times New Roman", Times, serif;
+  background: #333;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.main {
+  width: 1000px;
+  height: 400px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.row {
+  display: flex;
+  flex-direction: column;
+
+  height: 400px;
+  width: 300px;
+}
+
+.row-1 {
+  box-sizing: border-box;
+
+  position: relative;
+  height: 200px;
+  width: 300px;
+
+  border: 1px solid #333;
+  box-shadow: 0 20px 50px rgba(0, 0, 0, 0.8);
+
+  transform: translateY(100px);
+
+  z-index: 1;
+
+  background: #333;
+}
+
+.row:hover .row-1 {
+  background-color: #ff005f;
+
+  transition: 0.5s;
+  transform: translateY(0);
+}
+
+.row-1 h3 {
+  margin: 0 0 10% 40%;
+
+  font-size: 1.5rem;
+
+  opacity: 0.2;
+}
+
+.row:hover .row-1 h3,
+.row:hover .row-1 img {
+  opacity: 1;
+
+  transition: 0.5s;
+}
+
+.row-1 img {
+  padding: 10px;
+  margin-left: 30%;
+  margin-top: 4%;
+
+  opacity: 0.2;
+
+  text-align: center;
+}
+
+.row-2 {
+  height: 200px;
+  width: 300px;
```

Comment:
You should use grouping selector to group similar properties.

Reply:
ok @nasir-k